tools/ArduinoCsCompiler/samples/BlinkingLed/BlinkingLed.csproj

```diff
@@ -12,12 +12,15 @@
     <Compile Include="Program.cs" />
   </ItemGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition="$(Configuration)=='Debug'">
```

Comment:
I don't fully follow here why we use different P2Ps depending on the build config. Why can't we always just reference iot.device.bindings?

Reply:
Because the dependencies get really ugly for debugging and development in that case, as you would need to either link against the officially released package (which does not include the changes one might want to test) or rebuild the full package for every build. I've tried it, but it was impossible to work with.